doc/docs/Python_User_Interface.md

```diff
@@ -87,40 +87,40 @@ control various parameters of the Meep computation.
 
 ```python
 def __init__(self,
-             cell_size,
-             resolution,
-             geometry=None,
-             sources=None,
-             eps_averaging=True,
-             dimensions=3,
-             boundary_layers=None,
-             symmetries=None,
-             force_complex_fields=False,
-             default_material=Medium(),
-             m=0,
-             k_point=False,
+             cell_size:Union[meep.geom.Vector3, Tuple[float, ...], Tuple[int, ...]]=None,
+             resolution:Union[float, int]=None,
+             geometry:Union[List[meep.geom.GeometricObject], NoneType]=None,
+             sources:Union[List[meep.source.Source], NoneType]=None,
+             eps_averaging:bool=True,
+             dimensions:int=3,
+             boundary_layers:Union[List[meep.simulation.PML], NoneType]=None,
+             symmetries:Union[List[meep.simulation.Symmetry], NoneType]=None,
+             force_complex_fields:bool=False,
+             default_material:meep.geom.Medium=Medium(),
+             m:int=0,
```

Comment:
can also be `float`

Reply:
Fixed. Note that passing `int` when a `float` is expected is acceptable as discussed in https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type.